docroot/modules/custom/yukon_migrate/src/Plugin/migrate/source/YukonFilemime.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\yukon_migrate\Plugin\migrate\source;
+
+use Drupal\file\Plugin\migrate\source\d7\File;
+
+/**
+ * The 'yukon_migrate_file_mime' source plugin.
+ *
+ * @MigrateSource(
+ *   id = "yukon_migrate_file_mime",
+ *   source_module = "file"
+ * )
+ */
+class YukonFilemime extends File {
```

Comment:
We already have a custom source plugin for files - YukonFile. There is no point to create a new one to add this simple logic. Can you please move this source plugin logic to the YukonFile source plugin?

Reply:
This is now resolved.